charts/incubator/zerotier/values.yaml

```diff
@@ -0,0 +1,35 @@
+image:
+  repository: zerotier/zerotier
+  pullPolicy: IfNotPresent
+  tag: 1.8.4@sha256:b57e76e4291fbc26e1c75df07bc3534e9180457343c7ceff5d7ac8c4e25f6f44
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+  capabilities:
+    add:
+      - NET_ADMIN
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: UDP
+        targetPort: 9993
+        port: 9993
```

Comment:
We try to use incremental ports in the ports list, is 9993 required, or does the external port need to be the same as the internal port?

Reply:
I don't know, ports are not a must

Reply:
Can we try using a "port" number of the highest port in the incremental list and keep the "targetPort" at 9993?

Reply:
OK